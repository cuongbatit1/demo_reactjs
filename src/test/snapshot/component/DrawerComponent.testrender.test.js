import { render, screen } from "@testing-library/react";
import DrawerComponent from "../../../component/DrawerComponent";

test("test renders DrawerComponent snapshot", () => {
  const { asFragment } = render(<DrawerComponent />);
  expect(asFragment()).toMatchSnapshot();
});

test("test renders DrawerComponent snapshot with open true", () => {
  const { asFragment } = render(<DrawerComponent open={true} />);
  expect(asFragment()).toMatchSnapshot();
});
