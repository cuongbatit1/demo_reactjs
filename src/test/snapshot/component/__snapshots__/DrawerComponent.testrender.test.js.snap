// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders DrawerComponent snapshot 1`] = `<DocumentFragment />`;

exports[`test renders DrawerComponent snapshot with open true 1`] = `<DocumentFragment />`;
